src/index.css

```diff
@@ -0,0 +1,64 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;1,500&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  padding: 0;
+  color: rgb(255, 255, 255);
+  background-color: #cb4277;
+}
+
+.container {
+  display: grid;
+  justify-content: center;
+  flex-flow: wrap;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+img {
+  width: 100px;
+  height: 50px;
+  margin: auto;
+}
+
+.right-arrow-icon {
+  color: #fff;
+  margin-left: 80%;
+}
+
+.left-arrow-icon {
+  color: #fff;
+  margin-left: 2%;
+}
+
+.microphone-icon {
+  color: #fff;
+  margin-left: 90%;
+  margin-right: 10px;
+}
+
+.gear-icon {
+  color: #fff;
+}
+
+.city-card {
```

Comment:
- [ ] `required`
- [ ] Amazing job so far! 👏 
- [ ] However, there is just a few changes you missed from the previous reviewer's suggestions
- [ ] You took out the borderlines 👍  but you left out the colors patches 😞 
- [ ] Please see the screenshots below. and kindly make amends

`Your template`
![image](https://github.com/LDouglasOT/AirPollution/assets/66230306/d11e6602-2ce1-40d2-9e8d-93c20d09ad61)

`original template`
![image](https://github.com/LDouglasOT/AirPollution/assets/66230306/5927d5af-7282-4b4e-80ad-17420c392351)

Reply:
- [ ] As required by the previous reviewer, kindly follow the given design as it is a requirement. The previous reviewer advised that we give the cards alternating color patches for the cards so as to distinguish them from each other. Currently all your cards have the same color making it hard to distiguish them one from the other.
Try using the `CSS :nth-child() Selector` to achieve this. Checkout this resource [here](https://www.w3schools.com/cssref/sel_nth-child.php)


Your design
![image](https://github.com/LDouglasOT/AirPollution/assets/81932478/bc0aa52f-efe5-4091-8d14-ca789f4ba9f7)

Given design
![image](https://github.com/LDouglasOT/AirPollution/assets/81932478/8733f4e5-b081-48ba-aad8-cfa64632101e)